test/functional/fixtures/api/es-next/native-dialogs-handling/iframe/testcafe-fixtures/page-load-test.js

```diff
@@ -10,7 +9,8 @@ const iframeUrl = 'http://localhost:3000/fixtures/api/es-next/native-dialogs-han
 
 test('Expected dialogs after page load', async t => {
     await t
-        .setNativeDialogHandler(() => null);
+        .setNativeDialogHandler(() => null)
```

Comment:
We definitely should create an issue to support native dialogs invoked on page loading
Probably it should be something like a hook in addition to test controller's method